### PR TITLE
Add test for scheduler:VolumeCountConflicts

### PR DESCRIPTION
**Pull request by wgliang**
**What this PR does / why we need it**:
Add test for scheduler:VolumeCountConflicts

**Special notes for your reviewer**:

### CODE REVIEW

File: pkg/scheduler/algorithm/predicates/predicates_test.go [hunk 1]
```diff
@@ -1590,38 +1590,42 @@ func TestEBSVolumeCountConflicts(t *testing.T) {
...
 		},
 	}
-	ebsPVCPod := &v1.Pod{
-		Spec: v1.PodSpec{
-			Volumes: []v1.Volume{
-				{
-					VolumeSource: v1.VolumeSource{
-						PersistentVolumeClaim: &v1.PersistentVolumeClaimVolumeSource{
-							ClaimName: "someEBSVol",
+	onePVCPod := func(filterName string) *v1.Pod {
```
Comment (ravisantoshgudimetla) [inline pkg/scheduler/algorithm/predicates/predicates_test.go]: Can we refactor this higher level function so that it could be reused?

Comment (wgliang) [reply pkg/scheduler/algorithm/predicates/predicates_test.go]: done

File: pkg/scheduler/algorithm/predicates/predicates_test.go [hunk 2]
```diff
@@ -1578,46 +1578,52 @@ func TestServiceAffinity(t *testing.T) {
 	}
 }
 
-func TestEBSVolumeCountConflicts(t *testing.T) {
-	oneVolPod := &v1.Pod{
+func onePVCPod(filterName string) *v1.Pod {
```
Comment (ravisantoshgudimetla) [inline pkg/scheduler/algorithm/predicates/predicates_test.go]: nit: I think you can combine splitPVCPod and onePVCPod

Comment (wgliang) [reply pkg/scheduler/algorithm/predicates/predicates_test.go]: This is more clear. At present, it is only a change of the previous function form.

**Pull request merged**